website/server/api/modules.ts

```diff
@@ -21,9 +22,11 @@ async function fetchModuleStats (module) {
     ])
     module.downloads = npm.downloads.lastMonth
     module.stars = github.stars
+    module.updatedAt = +new Date(npm.updatedAt || undefined)
```

Comment:
I would suggest using `publishedAt` field to reflect `latest` tag

Reply:
Sorry what do you mean?

Reply:
```suggestion
    module.publishedAt= +new Date(npm.publishedAt || undefined)
```

updatedAt is from npm registry DB. Basically any change to registry changes that field which is less accurate if we want to sort by last version time

Reply:
Oh I misread - thanks!